templates/custom/media/media--image--teaser.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+/**
+ * @file
+ * Theme override to display a media item.
+ *
+ * Available variables:
+ * - name: Name of the media.
+ * - content: Media content.
+ *
+ * @see template_preprocess_media()
+ *
+ * @ingroup themeable
+ */
+#}
+<a{{ attributes }} href="{{ file_url(content.field_media_image.0['#item'].entity.uri.value) }}" aria-label="image caption" aria-describedby="image-gallery__open-gallery">
+  {{ title_suffix.contextual_links }}
+  {% if content %}
```

Comment:
If there is no content, you'll already have issues on the A element too.

Reply:
Mmm true.